qtawesome/_version.py

```diff
@@ -1,2 +1,2 @@
-version_info = (0, 1, 11)
+version_info = (0, 2, 0, 'dev')
```

Comment:
Should we use PEP 440 conventions for versions? I mean, in that case this should read `dev0` instead of `dev` (I think :-)